stopify-continuations/src/callcc/declVars.ts

```diff
@@ -30,9 +74,9 @@ const lift: Visitor = {
       if (decl.id.type !== 'Identifier') {
         throw new Error(`Destructuring assignment not supported`);
       }
-      const newDecl = t.variableDeclaration('var',
-                        [t.variableDeclarator(decl.id)]);
-      stmts.push(lifted(newDecl));
+
+      (fParent as TopDecl<bh.FunWithBody | t.Program>).decls!.push(lifted(decl.id));
```

Comment:
doesn't `lifted` need to take a `variableDeclaration`? This looks like it's just pushing an `identifier` to the top, not a decl, which seems wrong.

Reply:
It seems that lifted is no longer being used with the change. Apparently, when I `unshift` the `VariableDeclaration` on the top of the function, the visitor for `VariableDeclaration` doesn't get run on the newly added `VariableDeclaration`.

Removing `lifted` doesn't seem to break anything locally. If the above makes sense, I can simply delete the `lifted` code.